voicebox/node.py

```diff
@@ -150,5 +199,31 @@ def end_call(self, addr: str, inform_connection: bool = True):
     @staticmethod
     def validate_connection(address: str) -> bool:
 
-        return bool(input("Would you like to connect to this client at {}? ".format(address)))
+        return True

```

Comment:
The `validate_connection` method currently always returns `True`. If there's a need for actual validation logic in the future, consider implementing it or adding a `TODO` comment to indicate this. [medium]